packages/build/tests/core/tests.js

```diff
@@ -533,6 +533,36 @@ test.serial('configFileDirectories is passed to zip-it-and-ship-it', async (t) =
   ])
 })
 
+test.serial('internalSrcFolder is passed to zip-it-and-ship-it', async (t) => {
+  const zipItAndShipItSpy = sinon.spy(zipItAndShipIt, 'zipFunctions')
+
+  await new Fixture('./fixtures/functions_internal_src_folder').withFlags({ mode: 'buildbot' }).runWithBuild()
+  zipItAndShipItSpy.restore()
+  const { args: call1Args } = zipItAndShipItSpy.getCall(0)
```

Comment:
Do you want to assert the actual value that was passed? Might be good to check that it's a full path, for example.

Reply:
Yes! ✅